packages/grafana-ui/src/components/Badge/Badge.tsx

```diff
@@ -18,25 +18,28 @@ export interface BadgeProps extends HTMLAttributes<HTMLDivElement> {
   color: BadgeColor;
   icon?: IconName;
   tooltip?: string;
+  interactive?: boolean;
```

Comment:
We don’t really want to encourage more copyable content to be put in a tooltip which this prop might do :sweat_smile: 

If the error is critical it would be best not to put it in a tooltip (the Tooltip [documentation](https://grafana.com/developers/saga/components/tooltip) mentions that). We also alternatively have a [Toggletip](https://grafana.com/developers/saga/Components/toggletip) which should be used basically as a tooltip that can be interacted with.

These are some options that can be used as an alternative, but obviously we don’t want to dictate how the alerts UI looks, so if you want to achieve what this PR does, we would still prefer for the prop not to be aded to `Badge` (so this pattern is not encouraged), instead you could wrap your badge in a tooltip `<Tooltip><Badge /></Tooltip>`  and then in that case pass in the interactive prop, but it would be an exception so we are not opening a can of worms :smile: